crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -42,12 +39,20 @@ pub enum Error {
     MissingLastCommitHash,
 }
 
-/// Rollup data that relayer/conductor need to know.
-#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
-pub struct RollupData {
-    #[serde(with = "hex::serde")]
-    pub chain_id: Vec<u8>,
-    pub transactions: Vec<Vec<u8>>,
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct ChainId(#[serde(with = "hex::serde")] Vec<u8>);
```

Comment:
yeah I prefer `String` since I think it should be human-readable, but `SequenceAction::chain_id` is a `Vec<u8>` and since this is user-supplied it may not be a utf8 string. we should probably change that type to `String` also, if you agree I can open an issue